pkgs/development/python-modules/treeo/default.nix

```diff
@@ -8,21 +9,16 @@
 
 buildPythonPackage rec {
   pname = "treeo";
-  version = "0.4.0";
+  version = "unstable-2022-02-11";
```

Comment:
Given https://github.com/NixOS/nixpkgs/pull/159933 do we need this change in this PR?

Reply:
we can skip it in this pr if you'd prefer. I just wanted to make everything go green, but I'm happy if https://github.com/NixOS/nixpkgs/pull/159933 looks good to you

Reply:
Just left a comment on https://github.com/NixOS/nixpkgs/pull/159933, I suggest we unbundle `treeo` versioning issue and deal with it in that separate PR so that you can go ahead and merge this one?